packages.yml

```diff
@@ -8,47 +8,47 @@ packages:
   # - package: fivetran/snapchat_ads
   #   version: [">=0.5.0", "<0.6.0"]
     - git: https://github.com/fivetran/dbt_snapchat_ads.git
-      revision: MagicBot/dbt-utils-cross-db-migration
+      revision: feature/enable-null-urls
```

Comment:
reminder to switch revisions once all upstream packages have been merged with the migration branch!

Reply:
done